tests/ashley/machina_extensions/forum/test_forum_rename.py

```diff
@@ -0,0 +1,62 @@
+from django.test import TestCase
+from machina.apps.forum_permission.shortcuts import assign_perm
+from machina.core.db.models import get_model
+
+from ashley.factories import ForumFactory, LTIContextFactory, UserFactory
+
+Forum = get_model("forum", "Forum")  # pylint: disable=C0103
+
+
+class ForumRenameTestCase(TestCase):
+    """Test the rename admin feature of a forum"""
+
+    def setUp(self):
+        super().setUp()
+        self.user_factory = UserFactory
+        self.lti_context_factory = LTIContextFactory
+        self.forum_factory = ForumFactory
```

Comment:
I don't get the point of assigning these class names to an attribute as it's one line for a line?